jlm/hls/ir/hls.cpp

```diff
@@ -8,6 +8,8 @@
 namespace jlm::hls
 {
 
+size_t PointerSizeInBits = 64;
```

Comment:
Yeah no. Let's not introduce global variables. This should be handed in the respective methods as parameter.